src/qt/walletframe.cpp

```diff
@@ -4,17 +4,23 @@
 
 #include <qt/walletframe.h>
 
-#include <qt/overviewpage.h>
-#include <qt/walletmodel.h>
-#include <qt/walletview.h>
-
-#include <cassert>
-
+#include <QApplication>
+#include <QClipboard>
 #include <QGroupBox>
 #include <QHBoxLayout>
 #include <QLabel>
 #include <QPushButton>
 #include <QVBoxLayout>
+#include <cassert>
+#include <qt/guiutil.h>
+#include <qt/overviewpage.h>
+#include <qt/walletmodel.h>
+#include <qt/walletview.h>
+
+#include <node/ui_interface.h>
+#include <psbt.h>
+#include <qt/psbtoperationsdialog.h>
```

Comment:
Please keep the grouping of headers unchanged -- (1) our own headers, (2) C++ headers, (3) Qt headers.